messages/en.json

```diff
@@ -20,5 +20,7 @@
   "store/shipping-option-zipcode.shippinFilter.pickpu": "Pickup",
   "store/shipping-option-zipcode.updateButton.label": "Update",
   "store/shipping-option-zipcode.popoverButton.label": "Select a location",
-  "store/shipping-option-zipcode.popover.description": "Shipping options and speed may vary based on delivery address and store"
-}
\ No newline at end of file
+  "store/shipping-option-zipcode.popover.description": "Delivery options and delivery speeds may vary per address and store.",
```

Comment:
I find a bit weird to use "speed" in this context. I would rather use estimates or other similar word. Because isn't rate like delivery per day....

Reply:
Agree. @ArthurFerrao Is this just a boiler plate before the localization team review? or it was suggested by the localization team?